src/components/NewCampaignForm.vue

```diff
@@ -53,7 +48,7 @@ export default {
         return
       }
       const accounts = await web3.eth.getAccounts();
-      const factory = await Factory.at("0x7c70286f6991c660a0cC6d52A74aEBbDE45Da380");
+      const factory = await Factory.at(process.env.VUE_APP_FACTORY_ADDRESS);
```

Comment:
The factory address now lives in the .env file so we only have to change it in one place when deploying a new factory.